editor/components/default-block-appender/style.scss

```diff
@@ -0,0 +1,30 @@
+.editor-default-block-appender {
+	button {
+		border: none;
+		background: none;
+		box-shadow: none;
+		display: block;
+		width: 100%;
+		height: $text-editor-font-size * 6; // 1.5 times the height of an empty paragraph
```

Comment:
It's worth considering to introduce new scss variable `$empty-paragraph-height` to avoid those comments.